package com.transsion.member.view;

import android.content.Context;
import com.transsnet.loginapi.ILoginApi;
import kotlin.Metadata;
import kotlin.ResultKt;
import kotlin.Unit;
import kotlin.coroutines.Continuation;
import kotlin.coroutines.intrinsics.IntrinsicsKt;
import kotlin.coroutines.jvm.internal.DebugMetadata;
import kotlin.coroutines.jvm.internal.SuspendLambda;
import kotlin.jvm.functions.Function2;
import kotlin.jvm.internal.Intrinsics;
import kotlinx.coroutines.n0;

@Metadata(d1 = {"\u0000\f\n\u0002\u0018\u0002\n\u0002\u0010\u0002\n\u0002\b\u0002\u0010\u0002\u001a\u00020\u0001*\u00020\u0000H\n¢\u0006\u0004\b\u0002\u0010\u0003"}, d2 = {"Lkotlinx/coroutines/n0;", "", "<anonymous>", "(Lkotlinx/coroutines/n0;)V"}, k = 3, mv = {2, 1, 0})
@DebugMetadata(c = "com.transsion.member.view.MemberBasicView$startPay$1$success$1", f = "MemberBasicView.kt", l = {}, m = "invokeSuspend")
/* loaded from: classes5.dex */
final class MemberBasicView$startPay$1$success$1 extends SuspendLambda implements Function2<n0, Continuation<? super Unit>, Object> {
    int label;
    final /* synthetic */ MemberBasicView this$0;

    /* JADX INFO: Access modifiers changed from: package-private */
    /* JADX WARN: 'super' call moved to the top of the method (can break code semantics) */
    public MemberBasicView$startPay$1$success$1(MemberBasicView memberBasicView, Continuation<? super MemberBasicView$startPay$1$success$1> continuation) {
        super(2, continuation);
        this.this$0 = memberBasicView;
    }

    @Override // kotlin.coroutines.jvm.internal.BaseContinuationImpl
    public final Continuation<Unit> create(Object obj, Continuation<?> continuation) {
        return new MemberBasicView$startPay$1$success$1(this.this$0, continuation);
    }

    @Override // kotlin.jvm.functions.Function2
    public final Object invoke(n0 n0Var, Continuation<? super Unit> continuation) {
        return ((MemberBasicView$startPay$1$success$1) create(n0Var, continuation)).invokeSuspend(Unit.f67184a);
    }

    @Override // kotlin.coroutines.jvm.internal.BaseContinuationImpl
    public final Object invokeSuspend(Object obj) {
        ILoginApi mLoginApi;
        ILoginApi mLoginApi2;
        ILoginApi mLoginApi3;
        zm.f fVar;
        IntrinsicsKt.f();
        if (this.label != 0) {
            throw new IllegalStateException("call to 'resume' before 'invoke' with coroutine");
        }
        ResultKt.b(obj);
        mLoginApi = this.this$0.getMLoginApi();
        if (mLoginApi == null || !mLoginApi.a()) {
            mLoginApi2 = this.this$0.getMLoginApi();
            if (mLoginApi2 != null) {
                mLoginApi2.g(this.this$0);
            }
            mLoginApi3 = this.this$0.getMLoginApi();
            if (mLoginApi3 != null) {
                Context context = this.this$0.getContext();
                Intrinsics.g(context, "getContext(...)");
                mLoginApi3.j(context);
            }
        }
        fVar = this.this$0.purchaseListener;
        if (fVar != null) {
            fVar.b();
        }
        return Unit.f67184a;
    }
}
